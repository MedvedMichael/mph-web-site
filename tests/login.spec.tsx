import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LoginPage from '../pages/login'

test('should render', () => {
    const res = shallow(<LoginPage />)
    expect(toJson(res)).toMatchSnapshot()
})