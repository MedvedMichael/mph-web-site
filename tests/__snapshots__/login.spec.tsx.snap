// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div
    className="login-card card"
    style={
      Object {
        "margin": "10%",
        "padding": "2%",
      }
    }
  >
    <div>
      <h1>
        Sign in
      </h1>
      <div>
        <div>
          <input
            aria-label="Username"
            onChange={[Function]}
            placeholder="E-mail"
            type="text"
          />
        </div>
        <div>
          <input
            aria-label="Password"
            onChange={[Function]}
            placeholder="Password"
            type="password"
          />
        </div>
        <div>
          <button
            onClick={[Function]}
            type="submit"
          >
            Sign in
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
